### PR TITLE
[HDRP][DXR] Optmize HDRP_DXR_Tests to prevent crashes and reduced test time

### PR DESCRIPTION
**Pull request by remi-chapelain**
### Purpose of this PR
Try to fix crashes on Yamato by reducing load and footprint of DXR Tests. 
What's being done:
- Set a shared lighting settings for all HDRP_DXR_Project scenes with Baking disabled (save some memory footprint by not allowing some buffers)
- Reduce the number of lights of some tests
- Reduce the maximum screen space shadow from 24 to 10 (
- Change screen space shadow buffer format from R16 to R8 (Save memory footprint)
- Remove some useless gameobjects from some debug test (Save some shader variants)

image

---
### Testing status
- Launched all tests locally. All green ✔️ 
- ~~There's one test that fails, 902_Materials_SG_Variants_Unlit, but it also fails on master due to a problem with the Alpha To Coverage PR by @adrien-de-tocqueville that will be reverted?~~ **REVERTED**
- Launched [yamato](https://yamato.cds.internal.unity3d.com/jobs/902-

### CODE REVIEW

File: .yamato/_latest_editor_versions_trunk.metafile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 trunk:
-  updated_at_UTC: '2021-12-14 10:15:44.213485'
+  updated_at_UTC: '2022-01-11 00:04:15.704673'
```
Comment (sebastienlagarde) [inline .yamato/_latest_editor_versions_trunk.metafile:2]: please revert this change

Comment (remi-chapelain) [reply .yamato/_latest_editor_versions_trunk.metafile]: done.

**Comment by remi-chapelain** [comment]
Ran all the DXR Tests locally on master. All green ✔️ 
Ran all the DXR Tests locally on this branch. All green ✔️

**Review by sebastienlagarde — APPROVED** [review]
